test/it/parser/src/main/resources/case/dml/select-special-function.xml

```diff
@@ -3020,4 +3020,168 @@
+                        </parameter>
+                    </function>
+                </expr>
+            </expression-projection>
+        </projections>
+    </select>
+
+    <select sql-case-id="select_mod">
+        <projections start-index="7" stop-index="18">
+            <expression-projection start-index="7" stop-index="18" text="MOD(234, 10)">
+                <expr>
+                    <function function-name="MOD" text="MOD(234, 10)" start-index="7" stop-index="18">
+                        <parameter>
+                            <literal-expression value="234" start-index="11" stop-index="13" />
+                        </parameter>
+                        <parameter>
+                            <literal-expression value="10" start-index="16" stop-index="17" />
+                        </parameter>
+                    </function>
+                </expr>
+            </expression-projection>
+        </projections>
+    </select>
+
+    <select sql-case-id="select_month">
+        <projections start-index="7" stop-index="25">
+            <expression-projection start-index="7" stop-index="25" text="MONTH('2008-02-03')">
+                <expr>
+                    <function function-name="MONTH" text="MONTH('2008-02-03')" start-index="7" stop-index="25">
+                        <parameter>
+                            <literal-expression value="2008-02-03" start-index="13" stop-index="24" />
+                        </parameter>
+                    </function>
+                </expr>
+            </expression-projection>
+        </projections>
+    </select>
+
+    <select sql-case-id="select_monthname">
+        <projections start-index="7" stop-index="29">
+            <expression-projection start-index="7" stop-index="29" text="MONTHNAME('2008-02-03')">
+                <expr>
+                    <function function-name="MONTHNAME" text="MONTHNAME('2008-02-03')" start-index="7" stop-index="29">
+                        <parameter>
+                            <literal-expression value="2008-02-03" start-index="17" stop-index="28" />
+                        </parameter>
+                    </function>
+                </expr>
+            </expression-projection>
+        </projections>
+    </select>
+
+    <select sql-case-id="select_multilinestring">
+        <projections start-index="7" stop-index="73">
+            <expression-projection start-index="7" stop-index="73" text="ST_GeomFromText('MultiLineString((0 0, 1 1, 2 2),(3 3, 4 4, 5 5))')">
+                <expr>
+                    <function function-name="ST_GeomFromText" text="ST_GeomFromText('MultiLineString((0 0, 1 1, 2 2),(3 3, 4 4, 5 5))')" start-index="7" stop-index="73">
+                        <parameter>
+                            <literal-expression value="MultiLineString((0 0, 1 1, 2 2),(3 3, 4 4, 5 5))" start-index="23" stop-index="72" />
```

Comment:
Should MultiLineString also be parsed as a function?

Reply:
> Should MultiLineString also be parsed as a function?

`MultiLineString((0 0, 1 1, 2 2),(3 3, 4 4, 5 5))` is the string parameter of st_geomfromtext. How should I proceed with this?

Reply:
![image](https://github.com/user-attachments/assets/46130b64-d089-4e7d-8e46-d142ccab6316)

![image](https://github.com/user-attachments/assets/1e53d023-420f-4faa-b44e-5c4733cd91d4)
`(0 0, 1 1, 2 2)`  cannot be parse to `literal-expression`

Reply:
> ![image](https://private-user-images.githubusercontent.com/16489768/359773794-46130b64-d089-4e7d-8e46-d142ccab6316.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQzMDg4NTQsIm5iZiI6MTcyNDMwODU1NCwicGF0aCI6Ii8xNjQ4OTc2OC8zNTk3NzM3OTQtNDYxMzBiNjQtZDA4OS00ZTdkLThlNDYtZDE0MmNjYWI2MzE2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODIyVDA2MzU1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ2Y2RmYmE5OWJhM2FhYTQxZWY4ZWIxZTkyZDA2MmEzYTAyYTYyNzhkMGFjZmEyNjI0M2YzNjdkYjIwYmFiNjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.zQ-qWTjumJ46vYzImX1kdcXFLbOQ8bVSzlDuNj52FDw)
> 
> ![image](https://private-user-images.githubusercontent.com/16489768/359773000-1e53d023-420f-4faa-b44e-5c4733cd91d4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQzMDg4NTQsIm5iZiI6MTcyNDMwODU1NCwicGF0aCI6Ii8xNjQ4OTc2OC8zNTk3NzMwMDAtMWU1M2QwMjMtNDIwZi00ZmFhLWI0NGUtNWM0NzMzY2Q5MWQ0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODIyVDA2MzU1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWNhZjJmM2RmZDExNDkwYTc0NTQ0YmZjMDRhNzY3MzM4MDZmNzQ0OTQzNjA2Y2E2Y2FkNGJiNDE0YmQxZTVkYTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qy2NXLkf4GI9FiYESPq-_jqGdDyI2DSMtXBOJ1WjYN4) `(0 0, 1 1, 2 2)` cannot be parse to `literal-expression`

It looks great.